recipes-kernel/linux/linux-fslc-lts_6.1.bb

```diff
@@ -12,6 +12,8 @@ upstreaming in any form."
 
 require linux-imx.inc
 
+KERNEL_DEVICETREE_32BIT_COMPATIBILITY_UPDATE = "1"
```

Comment:
Hmm, I just tested if I could override this in local.conf and it didn't work. Maybe need to use `?=` here.

Reply:
Sure, I don't expect this to be overrideable inside `local.conf` as it's version-specific, not machine-specific.